examples/finnegans-wake-federated.py

```diff
@@ -40,29 +49,19 @@
 os.mkdir(CRUFTSPACE)
 os.mkdir(CERTIFICATE_DIR)
 
-# Read seed node metadata file
-try:
-    teacher_metadata_path = sys.argv[2]
-except IndexError:
-    raise ValueError("Missing path to seed node metadata file")
-else:
-    with open(teacher_metadata_path, "r") as f:
-        f.seek(0)
-        teacher_bytes = binascii.unhexlify(f.read())
-    URSULA = Ursula.from_bytes(teacher_bytes, federated_only=True)
-    print("Will learn from {}".format(URSULA))
-    URSULA.save_certificate_to_disk(CERTIFICATE_DIR)
-
+ursula_seed_node = SeednodeMetadata(checksum_address="0x154d9c2062a2Fd6f1a4eE827308634547ce84810",
```

Comment:
Yess!

Reply:
Heh - this has changed in both content and method since this commit; Nonetheless, :+1: